CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# Version 1.9.0
+
+- Re-introduce the thread-local task push optimization to the executor. (#93)
+- Bump `async-task` to v4.4.0. (#90)
+- Replace some unnecessary atomic operations with non-atomic operations. (#94)
+- Use weaker atomic orderings for notifications. (#95)
+- When spawning a future, avoid looking up the ID to assign to that future twice. (#96)
+- Avoid a panic when the runner's tick counter overflows. (#101)
```

Comment:
Since this is a fix for a bug introduced in #94 (in other words, the bug does not exist in any previous release), I don't think it needs to be mentioned in the changelog.
```suggestion
```